src/systemctl/systemctl.c

```diff
@@ -1279,7 +1301,11 @@ static int list_timers(int argc, char *argv[], void *userdata) {
 
         (void) pager_open(arg_pager_flags);
 
-        n = get_unit_list_recursive(bus, strv_skip(argv, 1), &unit_infos, &replies, &machines);
+        r = normalize_unit_names(strv_skip(argv, 1), ".timer", &timers_with_suffix);
+        if (r < 0)
+                return r;
+
+        n = get_unit_list_recursive(bus, timers_with_suffix, &unit_infos, &replies, &machines);
```

Comment:
Hmm, this makes this work:
```console
$ build/systemctl list-timers --no-{pager,legend} unbound-anchor 
Tue 2019-09-17 00:00:00 CEST  3h 15min left Mon 2019-09-16 10:02:49 CEST  10h ago unbound-anchor.timer unbound-anchor.service
```
but not this:
```console
$ build/systemctl list-timers --no-pager unbound-*
0 timers listed.
```
We have `expand_names()` that does globbing. I think `expand_names()` should be used here too. Always better to kill two birds with one stone.